### PR TITLE
hid-listen: init at 1.01

### PR DESCRIPTION
**Pull request by TomSmeets**
This tool has to be run as root.

taken from the [homepage](https://www.pjrc.com/teensy/hid_listen.html):
> On Ubuntu Linux, the /dev/hidraw device files are not readable by default for normal users. You could create a custom udev rule, or simply run hid_listen with sudo.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/hid-listen/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchzip {
+    name = "hid_listen_${version}";
+    url = "https://www.pjrc.com/teensy/hid_listen_${version}.zip";
+    sha256 = "0sd4dvi39fl4vy880mg531ryks5zglfz5mdyyqr7x6qv056ffx9w";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv ./hid_listen $out/bin/$hid_listen
```
Comment (bennofs) [inline pkgs/tools/misc/hid-listen/default.nix]: Why is `$hid_listen` a variable here? I don't follow

**Review by bennofs — COMMENTED** [review]
Just one small thing I don't quite understand, looks good otherwise! :+1:

File: pkgs/tools/misc/hid-listen/default.nix (hunk 1 above)
Comment (TomSmeets) [inline pkgs/tools/misc/hid-listen/default.nix]: Oh, that is a mistake.
nice catch!
I will fix this tomorrow.

Comment (TomSmeets) [inline pkgs/tools/misc/hid-listen/default.nix]: @bennofs Fixed with 75454c3122e230c4ad21582e6a48468e0fe9c62c

**Comment by bennofs** [comment]
I removed `darwin` from the supported platforms since I couldn't make it work. Will merge the PR after travis build finishes.

**Comment by TomSmeets** [comment]
Ok, thanks for merging.

**Pull request merged**